aerialist/ros_node_script/spawn_script/spawn_dynamic_obstacles.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+import subprocess
+import rospy
+import roslaunch
+
+
+def run_tree_subprocess(tree_name, x, y, z, r):
+    subprocess.Popen(
+        [
+            "python2",
+            "/src/catkin_ws/src/avoidance/spawn_obstacles/src/nodes/spawn_tree.py",
+            tree_name,
+            x,
+            y,
+            z,
+            r,
+        ]
+    )
+
+
+def run_box_subprocess(box_name, l, w, h, x, y, z, r):
+    subprocess.Popen(
+        [
+            "python2",
+            "/src/catkin_ws/src/avoidance/spawn_obstacles/src/nodes/spawn_box.py",
+            box_name,
+            l,
+            w,
+            h,
+            x,
+            y,
+            z,
+            r,
+        ]
+    )
+
+
+def run_apartment_subprocess(apartment_name, x, y, z, r):
+    subprocess.Popen(
+        [
+            "python2",
+            "/src/catkin_ws/src/avoidance/spawn_obstacles/src/nodes/spawn_apartment.py",
+            apartment_name,
+            x,
+            y,
+            z,
+            r,
+        ]
+    )
+
+
+def launch_node():
+    # Iterate over the array and launch nodes
+    rospy.init_node("dynamic_obstacle_launcher")
+    list1 = rospy.get_param("/dynamic_obstacle_launcher/obstacle_string").split(",end,")
```

Comment:
we could use a better delimitter. or better format. like: [,,,,],[,,,],[,,,,],[,,,,]. let's discuss this